server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CoordinatorExecutable.java

```diff
@@ -42,6 +42,7 @@ public String description() {
 
   @Override
   public void execute(final String[] args) throws Exception {
+    System.out.println("WARNING: External compaction processes are experimental");
```

Comment:
System.err or using the logger might be better for these.

Reply:
I'll change to `err`. My intent was for something to go to .out and not get lost with all of the other output in the log file. Using `.err` may be more noticeable to the user.

Reply:
Changed to `err` in 42d0c8445d964f079f39ef4c50d05559191982a3